packages/terra-list/src/ListSection.jsx

```diff
@@ -22,9 +22,9 @@ const propTypes = {
    */
   isCollapsible: PropTypes.bool,
   /**
-   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`.
+   * Optionally sets the heading level. One of `2`, `3`, `4`, `5`, `6`.
    */
-  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
+  level: PropTypes.oneOf([2, 3, 4, 5, 6]),
```

Comment:
is there a reason we are removing default header as one ?

Reply:
As per UX recommendation level 1 should be removed. So to make it passive I have internally converted the h1 to h2 and reverted this change and added apt descriprtion